src/utils/hearing.js

```diff
@@ -47,6 +47,14 @@ export function acceptsComments(hearing) {
   return !hearing.closed && (new Date() < new Date(hearing.close_at));
 }
 
+/*
+* Returns true if any of the sections in a hearing has commenting enabled or if the hearing has a plugin
+* else false.
+ */
+export function hasCommentableSections(hearing, sections, user) {
```

Comment:
What is this function used for? Couldn't find it anywhere else.

Reply:
It's used here https://github.com/City-of-Helsinki/kerrokantasi-ui/pull/719/commits/cf715a28bab9e861791d21b7acb9464b97b16f80#diff-abd7d8933838e02a52669dd5b27b93cbR85